antsibull/write_docs.py

```diff
@@ -293,10 +311,14 @@ async def output_indexes(collection_info: CollectionInfoT,
     :arg squash_hierarchy: If set to ``True``, no directory hierarchy will be used.
                            Undefined behavior if documentation for multiple collections are
                            created.
+    :arg collection_infos: Optional dictionary mapping collection names to collection info objects
```

Comment:
Oh, I think the docstring for collection_info is wrong (preexisting).  I think it should read:
```
    :arg collection_info: Mapping of collection_name to Mapping of plugin_type to Mapping of
        plugin_name to short_description.
```

I bring it up in this PR since I sent myself down a rabbithole when trying to think about this PR with the incorrect docstring in my mind.